shared/js/ui/views/internal-options.js

```diff
@@ -36,11 +45,58 @@ export default function InternalOptionsView (ops) {
     this.template = ops.template = template
     this.pageView = ops.pageView
     ViewParent.call(this, ops)
-    this.model.getState().then(() => {
-        this._rerender()
-    })
+    this.reload()
 }
 
 InternalOptionsView.prototype = window.$.extend({}, ViewParent.prototype, {
-
+    setup: function () {
+        this._cacheElems('.js-options', ['config-url', 'set-config-url'])
+        this.bindEvents([
+            [this.$setconfigurl, 'click', this._clickSetting],
+            [this.$configurl, 'input', this._onURLChange]
+        ])
```

Comment:
Should we align with other implementations, and support things like 'reload now' <- I found myself wanting to do that when testing :)

<img width="912" alt="image" src="https://github.com/duckduckgo/duckduckgo-privacy-extension/assets/1643522/4ffbfd7f-7795-4785-b7e6-41f24401cbf7">

Reply:
The config should reload automatically when using the new debugger tool, but for consistency with other platforms I've added this by adding another state to the 'set'/'clear' button.

Reply:
> The config should reload automatically when using the new debugger tool, but for consistency with other platforms I've added this by adding another state to the 'set'/'clear' button.


Ahh that's good then - I think it's a nice change because then this is useful outside of the new debugger interface too :)